services/self-service/src/main/java/com/epam/dlab/backendapi/dao/EndpointDAOImpl.java

```diff
@@ -65,11 +74,23 @@ public void remove(String name) {
 		deleteOne(ENDPOINTS_COLLECTION, endpointCondition(name));
 	}
 
+	/**
+	 * According the functional requirement, the EndPoint name should be case-insensitive.
+	 * A New method use the regex interface of the MongoDB driver
+	 */
+	@Deprecated //private Bson endpointCondition(String name) { return eq(ENDPOINT_NAME_FIELD, name); }
+
 	private Bson endpointCondition(String name) {
-		return eq(ENDPOINT_NAME_FIELD, name);
+		//The Pattern interface is used, to avoid case sensitive endpoint Name
```

Comment:
Please do not you any comment inside java code)

Reply:
Sure)